### ISSUE TITLE
FR: Display a popup note when hovering over a dangling link

### ISSUE THREAD

**Issue by Taitava** [graydon/obsidian-dangling-links#19]
My screenshots are in Finnish, but I guess they can still show the idea.

I have the _Dangling links_ panel open here:

![kuva](https://user-images.githubusercontent.com/13002142/173243898-29dc3ed2-2180-48ea-b825-e57962977f7a.png)

If I hover over the first _Väri_ link (points to _Unelma.md_), I'd like to see a similar popup that I can get in core Obsidian when hovering over a link in a note (and pressing `Ctrl`/`Cmd` at the same time):

![kuva](https://user-images.githubusercontent.com/13002142/173244055-0ef6a595-8919-44e6-961b-895e1a4de589.png)

The popup view could scroll to and highlight the dangling link in the note.

Please let me know if there's anything unclear, I wrote this quite quickly 🙂. Thank you for this nice plugin! 😎

**Comment by claremacrae** [comment]
Hi @Taitava,

I think that you are saying that if you hover over a link in the Dangling Links pane, you would like to see the context of the dangling link, that is, the location of the dangling link.

Have I understood correctly?

**Comment by Taitava** [comment]
Yes, I think that's what I meant - the location of the dangling link. I had already forgotten this request, but I feel like showing the container page of the link seems reasonable. So:
- When hovering over the first _Väri_ link, the popup could show the page _Unelma_
- When hovering over the second or third _Väri_ link, the popup could show the page _Kauneus_.

Btw. would it be possible for the popup to be automatically scrolled to the place where the link resides? 🙂 It's not mandatory, but could make this even more useful. 👍 

Thank you for considering this feature! 🌞

**Comment by claremacrae** [comment]
Thank you for the reply.

> Btw. would it be possible for the popup to be automatically scrolled to the place where the link resides?

It would certainly be nice - it's almost likely to be possible... What remains is to figure out how to do it, and how long that will take.

Just to set expectations, my only priority right now is to make sure that all the open requests on this project are clear, precise, and request only one thing! 😄 

Hence all the issues labelled `question` currently.

I have renamed this to make the title a little more precise.

I will remove the `question` label from this one now. Thanks again.

**Comment by claremacrae** [comment]
PS 3 of the 4 remaining questions are on issues created by yourself!
